package.json

```diff
@@ -428,6 +428,12 @@
 						"markdownDescription": "%asciidoc.use_asciidoctorpdf.desc%",
 						"markdownDeprecationMessage": "%asciidoc.use_asciidoctorpdf.deprecationMessage%",
 						"scope": "resource"
+					},
+					"asciidoc.editor.drop.enabled": {
```

Comment:
I think we should move it into a dedicated group "Editor" since it's not related to the "Preview".
Also, we should name it `asciidoc.editor.enableDrop` to be consistent with other options.